ERCS/erc-6538.md

```diff
@@ -22,70 +22,241 @@ The standardization of stealth address generation holds the potential to greatly
 
 The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
 
-This contract defines an `ERC5564Registry` that stores the stealth meta-address for entities. These entities may be identified by an address, ENS name, or other identifier. This MUST be a singleton contract, with one instance per chain.
+This contract defines an `ERC6538Registry` that stores the stealth meta-address for entities. These entities may be identified by an address, ENS name, or other identifier. This MUST be a singleton contract, with one instance per chain.
 
 The contract is specified below. A one byte integer is used to identify the stealth address scheme. This integer is used to differentiate between different stealth address schemes. A mapping from the scheme ID to it's specification is maintained at [this](../assets/eip-5564/scheme_ids.md) location.
```

Comment:
Once the EIP goes to final you wont be able to update this list. EIPs aren't a great place to store this information.

Reply:
Updated in https://github.com/ethereum/ERCs/pull/259/commits/f7a4aec74220455e12006d99ce1fae7b94655bbc , but please let me know if you have any recommendations for alternative approaches to this.